model/src/school/courses.ts

```diff
@@ -30,9 +21,6 @@ export interface Course {
 	periods: CourseTimePeriod[];
 	section: Section;
 	credits: Credits;
-	newAssignments: boolean;
```

Comment:
Same as before : How do you plan to handle assignments, including when new ones are added and when the course has started so that students can access them, if you decide to remove these fields: newAssignments, assignments, and started?